roles/openshift_openstack/defaults/main.yml

```diff
@@ -249,6 +254,11 @@ openshift_openstack_cns_secgroup_rules:
     protocol: tcp
     port_range_min: 24008
     port_range_max: 24008
+  # glusterblockd
+  - direction: ingress
+    protocol: tcp
+    port_range_min: 24010
+    port_range_max: 24010
```

Comment:
Is this due to rebase? I don't think changes to this file are relevant to the PR.

Reply:
Yes, bad rebase, reverted